src/bootstrap/src/core/builder/mod.rs

```diff
@@ -1605,6 +1605,13 @@ Alternatively, you can set `build.local-rebuild=true` and use a stage0 compiler
         }
     }
 
+    /// Gets a command to run the compiler specified.
```

Comment:
Please add a short mention of the lib path behaviour in this doc comment, so that it's easier to spot when hovering over the call in IDEs.

Reply:
Done!